src/intTest/java/uk/nhs/digital/nhsconnect/nhais/jms/InboundQueueServiceRegistrationTest.java

```diff
@@ -77,14 +81,12 @@ void whenMeshInboundQueueRegistrationMessageIsReceived_thenMessageIsHandled(Soft
         assertOutboundQueueRecepMessage(softly);
     }
 
-    private void assertOutboundQueueRecepMessage(SoftAssertions softly) throws JMSException, IOException {
-        var outboundQueueMessage = getOutboundQueueMessage();
-
-        var meshMessage = parseOutboundMessage(outboundQueueMessage);
+    private void assertOutboundQueueRecepMessage(SoftAssertions softly) throws IOException {
+        List<String> msgs = meshClient.getInboxMessageIds();
+        var meshMessage = meshClient.getEdifactMessage(msgs.get(0));
 
         softly.assertThat(meshMessage.getContent()).isEqualTo(new String(Files.readAllBytes(recep.getFile().toPath())));
         softly.assertThat(meshMessage.getWorkflowId()).isEqualTo(WorkflowId.RECEP);
-        softly.assertThat(meshMessage.getMessageSentTimestamp()).isNotNull();
```

Comment:
Timestamp is null now

Reply:
Method where this takes place should be renamed to not mention queue